datasets/basiskaart/dataset.json

```diff
@@ -341,6 +341,53 @@
         }
       }
     },
+    {
+      "id": "waterdeelvlak",
+      "title": "Waterdeel vlakobject",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding van het record.",
+            "provenance": "identificatie_lokaalid"
+          },
+          "geometrie": {
```

Comment:
Als het geometrie veld niet `geometry` heet, is het de bedoeling het in het schema (naast `display`) op te nemen als `mainGeometry`. Dus hier `"mainGeometry": "geometrie"`.

Reply:
Ah didn't know. Thanks for pointing it out.

Fixed it, and for all existing basiskaart tables as well.